src/screens/dashboard/DashboardScreen.tsx

```diff
@@ -28,12 +28,21 @@ export class DashboardScreen extends React.PureComponent<NavigationInjectedProps
   render() {
     const student = this.props.navigation.getParam('student');
 
+    const navigateToCreatePlan = () =>
```

Comment:
You should create this function outside of the `render` method, inside the component class scope.

Reply:
fixed